### ISSUE TITLE
default value for expanded constructor option is 'false'

### ISSUE THREAD

**Issue by jgravois** [Esri/esri-leaflet-geocoder#70]
we just need to update to the README (it currently says `true`)

cc @benstoltz

**Comment by jgravois** [comment]
might as well bump to beta.4 in the code snippet while we're at it
```js
<script src="http://cdn-geoweb.s3.amazonaws.com/esri-leaflet-geocoder/0.0.1-beta.3/esri-leaflet-geocoder.js"></script>
```

**Comment by jgravois** [comment]
closed by #71.  thanks @benstoltz!
